README.md

```diff
@@ -18,4 +20,167 @@ Execute the following steps to practice various topics covered in the intro to g
 12) Now merge the other branch with main branch using git merge myFirstBranch <br/>
 13) Now push all the changes onto your forked github repository using git push origin/main <br/>
  
- Open your forked github repository and check whether all the changes have been updated. <br/>
\ No newline at end of file
+ Open your forked github repository and check whether all the changes have been updated. <br/>
+
+## Log
+
+`git log` is used to display the commit history of the repository. Try using this command and you will be able to see the commit history of the repository.
+
+```bash
+git log
+```
+
+However, the output of the above command may be too verbose. You can use the following command to see a more concise output.
+
+```bash
+git log --all --decorate --oneline --graph
+```
+
+In the current state, you should see a commit history that looks like this:
+
+![Commit Log](/content/images/log_mid.png)
```

Comment:
The commit messages are a bit misleading, could you change it such that they reflect which commit comes first? For example the last one maybe named "My Latest Commit" etc.

Reply:
Is it about `commit1` and `commit2` which are present in all the images? I guess I can simply not show that as it is not relevant